_posts/2021-04-22-introducing-ugrcodes.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Introducing UGRCodes"
+author:
+  display_name: Erik Neemann
+  email: eneemann@utah.gov
+date: 2021-04-22 15:30:12
+categories:
+  - SGID Blog
+  - Featured
+tags:
+  - sgid
+---
+
+Are you tired of fighting with esoteric municipal FIPS codes? Have you even tried using them? It's not easy and it's certainly not intuitive. Five digit integer codes with minimal consistency and continuity across space, and require a lookup table to understand, don't lend themselves as a simple identifier for a municipality. As a solution, UGRC is implementing a new attribute field called the UGRCode (get it?) in our [Municipal Boundaries](https://opendata.gis.utah.gov/datasets/utah-municipal-boundaries) and [Metro Townships](https://opendata.gis.utah.gov/datasets/utah-metro-townships) data. The full list of UGRCodes can be found [here](https://docs.google.com/spreadsheets/d/19cwbXoewTynDDv_fJ3hGekD4VbLef5VxrnSJfRx8My8/edit#gid=2028111876).
+
+[![UGRCodes in Salt Lake County]({% link images/20210422_ugrcodes.png %} "UGRCodes in Salt Lake County")]({% link images/20210422_ugrcodes.png %}){: .flex .flex--center }
+
+Inspired by a suggestion from one of our colleagues at the Wasatch Front Regional Council, we believe these codes provide several advantages that make them more useful and easier to use than FIPS codes or municipality names for uniquely identifying municipalities in Utah. They also support our efforts to ensure that attribute domain values match their corresponding descriptions within our datasets. Some advantages of the UGRCodes are:

```

Comment:
**thought**: it would be nice to link to the process we have for this since it is a little nebulous here.

> They also support our efforts to ensure that attribute domain values match their corresponding descriptions within our datasets

Reply:
Done